pyserini/prebuilt_index_info.py

```diff
@@ -6195,4 +6614,5 @@ def add_lucene_index_info(enum, info, name=None, readme=None):
                     **FAISS_INDEX_INFO_MIRACL,
                     **FAISS_INDEX_INFO_WIKIPEDIA,
                     **FAISS_INDEX_INFO_CIRAL,
+                    **FAISS_INDEX_INFO_MBEIR,
```

Comment:
Please change this to `FAISS_INDEX_INFO_M_BEIR` so it's consistent with everything else.